gradle/verification-metadata.xml

```diff
@@ -881,6 +883,11 @@
             <pgp value="5b131e826582cf79510daa11cd3e539f208832d0"/>
          </artifact>
       </component>
+      <component group="net.openhft" name="zero-allocation-hashing" version="0.10.1">
```

Comment:
This entry is not at the right position. Follow instructions in `gradle/verification-metadata.xml` to obtain a stable output.

Reply:
Now I finally read the README there :sweat_smile: Thanks! It should be correct now.